dashboard/app/reporting_email.go

```diff
@@ -513,15 +513,15 @@ var sendEmail = func(c context.Context, msg *aemail.Message) error {
 	return nil
 }
 
-func ownEmail(c context.Context) string {
+var ownEmail = func(c context.Context) string {
```

Comment:
It would be cleaner to put these into config explicitly and if not set use the current code. I assume you need statically known app name/email, right?

Reply:
Right, i already have my custom values in `config_prod.go` and thought i should do it like `sendEmail`. What is your suggestion ?

Reply:
The function mocking is intended for testing. For production is better to have config values.
These are a more explicit part of "public interface" and, say, if I need to change the function signature in future, it will be harder with external mocking.

Reply:
ok, i'll move it into dashboard config and push again, thanks for feedback